alchemist/alchemist-cognitive-agents/src/main/resources/it/unibo/alchemist/model/cognitiveagents/characteristics/parameters.toml

```diff
@@ -0,0 +1,112 @@
+[cognitive]
+sensingOmega = 0.5
+affectiveBiasingOmega = 1.0
+persistingOmega = 0.95
+amplifyingFeelingOmega = 1.0
+inhibitingFeelingOmega = 1.0
+amplifyingEvacuationOmega = 1.0
+inhibitingWalkRandOmega = -1.0
+amplifyingIntentionOmega = 1.0
+inhibitingIntentionOmega = -1.0
+mentalEta = 0.9
+bodyEta = 0.25
+logisticSigma = 20.0
+logisticTau = 0.5
+advancedLogisticSigma = 2.0
+advancedLogisticTau = 0.14
+
+[speed]
+childMale = 0.5
+adultMale = 1.0
+elderlyMale = 0.9
+childFemale = 0.5
+adultFemale = 0.9
+elderlyFemale = 0.9
+default = 1.0
+variance = 0.5
+
+[compliance]
+childMale = 0.89
+adultMale = 0.89
+elderlyMale = 0.92
+childFemale = 0.89
+adultFemale = 0.94
+elderlyFemale = 0.97
+
+[helpAttitude]
+    [helpAttitude.adultMale]
```

Comment:
Do you need to explicitly declare supersections? I believe you don't. Can you try?

Reply:
True, I can omit them